### PR TITLE
Actualize helper

### PR DESCRIPTION
**Pull request by leonid-sviderskii-mtl**
I had doubts about having either 2 separate methods (`I.eyeCheck()` and `I.eyeCheckRegion()`) with some duplicated code or add the region logic to the `I.eyeCheck()`one.

So please review the edits and make changes if needed!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -58,6 +58,30 @@ class ApplitoolsHelper extends Helper {
...
     }
+
+    /**
+    * @param pageName {String} name of the page you want to check
+    * @param element {String} target element on the page that Applitools will focus on
+    * @param uniqueId {String} provide a unique id to combine tests into a batch
+    * @param matchLevel {String} set the match level. Possible values: Exact, Strict, Content, Layout
+    * 
+     */
+    async eyeCheckRegion(pageName, element, uniqueId, matchLevel) {
```
Comment (PeterNgTr) [inline index.js:69]: aha, so we do could something like this

`
async eyeCheck({ pageName, element, uniqueId, matchLevel }) {
 ...
         await eyes.open(client, appName, pageName, windowsSize);
         element ? await eyes.check(pageName, Target.region(element)) : await eyes.check(pageName, Target.window());
         await eyes.close();
}
`

Comment (leonid-sviderskii-mtl) [reply index.js:69]: My only question is about the `eyes.setForceFullPageScreenshot();` which I guess needs to be false for `Target.region` and true for `Target.window`. If so, the solution won't be that elegant but should still work. What do you think?

```
    async eyeCheck({ pageName, element, uniqueId, matchLevel }) {
        eyes.setApiKey(this.config.applitoolsKey);
        eyes.setForceFullPageScreenshot(TRUE/FALSE);
        
        if (uniqueId) {
            eyes.setBatch(pageName, uniqueId);
        }

        if (matchLevel) {
            eyes.setMatchLevel(matchLevel);
        }

        await eyes.open(client, appName, pageName, windowsSize);
        element ? await eyes.check(pageName, Target.region(element)) : await eyes.check(pageName, Target.window()); 
        await eyes.close(); 
    }
```

Comment (leonid-sviderskii-mtl) [reply index.js:69]: I updated the code in the comment above

Comment (PeterNgTr) [reply index.js:69]: I think it's okay to go :D

Comment (leonid-sviderskii-mtl) [reply index.js:69]: I got stuck, to be honest. Using the code from above returns this error from the Applitools SDK: `IllegalArgument: testName is null or undefined`. I tried to play with the `eyes.setForceFullPageScreenshot()` as that's the only difference in the order of the commands but no luck.

I can propose 2 options:
1. If you have time, can you please take a look?
2. If not, maybe, we can merge it like it was before with 2 separate methods?

**Comment by leonid-sviderskii-mtl** [comment]
@PeterNgTr please take a look at Readme. As selector is the 2nd parameter and it's optional, this is how it will look like if we want to take a full-page screenshot and provide unique id and/or match level `I.eyeCheck('Homepage', undefined, 'Unique ID', 'Layout');`

Are you ok or any suggestions?

File: index.js [hunk 2]
```diff
@@ -39,23 +39,32 @@ class ApplitoolsHelper extends Helper {
...
     /**
     * @param pageName {String} name of the page you want to check
+    * @param element {String} selector of the target element which will be used as area for screenshot
     * @param uniqueId {String} provide a unique id to combine tests into a batch
-    * @param matchLevel {String} set the match level. Possible values: Extract, Strict, Content, Layout
+    * @param matchLevel {String} set the match level. Possible values: Exact, Strict, Content, Layout
     * 
      */
-    async eyeCheck(pageName, uniqueId, matchLevel) {
+    async eyeCheck(pageName, element, uniqueId, matchLevel) {
```
Comment (PeterNgTr) [inline index.js:47]:
```suggestion
    async eyeCheck({ pageName, element, uniqueId, matchLevel }) {
```

File: readme.md [hunk 3]
```diff
@@ -59,23 +59,32 @@ const { I } = inject();
...
     await I.eyeCheck('Homepage');
 });
 
-// Provide the second param to gather multiple tests
+// To be able to focus just on a single element instead of the full page, provide the element selector 
+// more info could be found here https://www.npmjs.com/package/@applitools/eyes-webdriverio#region-screenshot
+Scenario('Check a CTA button on home page @test', async () => {
+    const targetElement = 'div.section.button-section > button';
+    I.amOnPage('https://applitools.com/helloworld');
+    await I.eyeCheck('CTA button', targetElement);
```
Comment (PeterNgTr) [inline readme.md:73]:
```suggestion
    await I.eyeCheck({ pageName: 'CTA button', element: targetElement});
```

File: readme.md [hunk 4]
```diff
@@ -59,23 +59,32 @@ const { I } = inject();
...
+});
+
+// Provide the third param to gather multiple tests
 // more info could be found here https://help.applitools.com/hc/en-us/articles/360006914772-Batching
 Scenario('Check home page @test', async () => {
     I.amOnPage('https://applitools.com/helloworld');
-    await I.eyeCheck('Homepage', 'Layout');
+    await I.eyeCheck('Homepage', undefined, 'Unique ID');
```
Comment (PeterNgTr) [inline readme.md:80]:
```suggestion
    await I.eyeCheck({ pageName: 'Homepage', uniqueId: 'Unique ID' });
```

File: readme.md [hunk 5]
```diff
@@ -59,23 +59,32 @@ const { I } = inject();
...
 });
 
-// Provide the third param to set the match level
+// Provide the fourth param to set the match level
 // more info could be found here https://help.applitools.com/hc/en-us/articles/360007188591-Match-Levels
 Scenario('Check home page @test', async () => {
     I.amOnPage('https://applitools.com/helloworld');
-    await I.eyeCheck('Homepage', 'Homepage', 'Layout');
+    await I.eyeCheck('Homepage', undefined, 'Unique ID', 'Layout');
```
Comment (PeterNgTr) [inline readme.md:87]:
```suggestion
    await I.eyeCheck({ pageName: 'Homepage', uniqueId: 'Unique ID', matchLevel: 'Layout');
```

**Review by PeterNgTr — COMMENTED** [review]
We can pass the parameter as object then we don't need to care the optional param

**Comment by leonid-sviderskii-mtl** [comment]
> We can pass the parameter as object then we don't need to care the optional param

I got stuck at some point :) My tests are working on desktop but not on mobile devices via Browserstack. I'm in contact with Applitools' support team so in case we need to make any changes to the PR, I'll just leave it open for now.

**Comment by PeterNgTr** [comment]
I'll plan to update the core, so merging for now.

**Pull request merged**